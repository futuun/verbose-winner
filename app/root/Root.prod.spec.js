import React from 'react'
import renderer from 'react-test-renderer'
import configureStore from '../store'
import Root from './Root.prod'

const store = configureStore()

test('Root: initialize app', () => {
  fetch.mockResponse(JSON.stringify({ word: 'kopytko' }))

  const component = renderer.create(
    <Root store={store} />
  )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
