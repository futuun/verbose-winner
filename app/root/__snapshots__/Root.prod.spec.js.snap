exports[`test Root: initialize app 1`] = `
<div
  style={
    Object {
      "height": "100%",
    }
  }>
  <div
    className="game">
    <div
      className="board">
      <div
        className="missed">
        []
        <br />
        \"\"
        <br />
        []
        <br />
      </div>
    </div>
  </div>
</div>
`;
